spec/compilers/css_definition

```diff
@@ -13,7 +13,7 @@ component Main {
 }
 --------------------------------------------------------------------------------
 class A extends _C {
-  $a() {
+  $aa() {
```

Comment:
All of the `aa` will be removed and changed with a prefix.

I keep them to simplify a grep :)